gtk-vnc.rb

```diff
@@ -0,0 +1,76 @@
+class GtkVnc < Formula
+  desc "VNC viewer widget for GTK."
+  homepage "https://wiki.gnome.org/Projects/gtk-vnc"
+  url "https://download.gnome.org/sources/gtk-vnc/0.5/gtk-vnc-0.5.4.tar.xz"
+  sha256 "488aa97a76ce6868160699cd45d4a0ee0fe6f0ad4631737c6ddd84450f6c9ce7"
+
+  depends_on "gobject-introspection" => :optional
+  depends_on "gettext" => :build
+  depends_on "gdk-pixbuf"
+  depends_on "gnutls"
+  depends_on "gtk+3"
+  depends_on "intltool" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pulseaudio" => :optional
+  depends_on "vala" => :optional
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!